tuichain/api/views/loans.py

```diff
@@ -13,7 +13,10 @@
 from rest_framework.permissions import *
 from rest_framework.decorators import api_view, permission_classes
 import decimal
+import statistics
```

Comment:
Can remove this line and replace all occurrences of `statistics.StatisticsError` by `StatisticsError`.